include/graphit/midend/mir.h

```diff
@@ -857,6 +859,7 @@ namespace graphit {
         struct ApplyExpr : public Expr {
             Expr::Ptr target;
             std::string input_function_name = "";
+            std::vector<std::string> functorArgs;
```

Comment:
Why are all these strings? They should be `std::vector<mir::Expr>` so that they can be handled by code generator and analyses in a clean way.

Reply:
I certainly can change them back to mir::Expr. I initially had them mir::Expr for ApplyExpr but apparently edgeset generation requires FROM and TO functor args to be vector of strings because there was a helper function that generates functor call as string -> https://github.com/GraphIt-DSL/graphit/blob/master/src/backend/codegen_cpp.cpp#L1581. If the functor args for FROM and TO were mir::Expr, it would have required more code changes so I was reluctant to change them to mir::Expr and decided to leave them as strings.  
And  I thought it would be more consistent to just have all the mention of functorArgs . Let me know what you think?

Reply:
Yeah, it is legacy of a hack I did before (forcing the arguments to be strings). For now, I think this is fine.

Reply:
@yunmingzhang17 do you think we should also make it FuncExpr as well? Since we are doing it for EdgeSetExpr, might as well we do it for ApplyExpr to be consistent

Reply:
yup, if you make a MIR FuncExpr, then this might be a good idea.